app/models/ucsd_agent.rb

```diff
@@ -0,0 +1,30 @@
+# UcsdAgent is a subclass of EDM Agent (http://www.europeana.eu/schemas/edm/Agent)
+class UcsdAgent < Agent
+
+  configure type: ::UcsdTerms.UcsdAgent
+
+  property :agent_type, predicate: ::UcsdTerms.agent_type, multiple: false
+  property :has_orcid, predicate: ::RDF::URI.new("http://id.loc.gov/vocabulary/identifiers#orcid"), multiple: false
```

Comment:
Did @ucsdlib/domm decide the property would be called `has_orcid`? That seems strange to me, as it reads more like a boolean method name or conditional check. Could we just use `orcid`?